docs/UserGuide.md

```diff
@@ -67,6 +67,10 @@ and flags(e.g. `-p`, `-D`, etc.) are **case-sensitive**.
 Adds an item to the list of tracked inventory.
 
 Format: `addi n/ITEM_NAME q/QUANTITY d/DESCRIPTION [t/TAG]…​ sp/SELL_PRICE cp/COST_PRICE`
+- `SELL_PRICE` is the price that the user sells the item for
```

Comment:
Actually, I was thinking that since tags are optional, it might be better to move tags to the back of the format, and bring `sp` and `cp` forward. So format would look like this:

`Format: `addi n/ITEM_NAME q/QUANTITY d/DESCRIPTION sp/SELL_PRICE cp/COST_PRICE`  [t/TAG]…​`

Might also work to change the `...` to something else because it might look ambiguous, since `...` isn't actually part of the command format

Reply:
Also for `SELL_PRICE`, I think a better description would be

`SELL_PRICE` is the amount that is received as revenue per unit of the item sold